postman/collection-dir/payout/#2 Payout in stages - create + confirm + fulfill API calls/Bank payouts/Create SEPA Payout/request.json

```diff
@@ -0,0 +1,75 @@
+{
+  "method": "POST",
+  "header": [
+    {
+      "key": "x-feature",
+      "value": "integ-custom",
+      "type": "text"
+    }
```

Comment:
here as well. we've this part code in many places down the line, please look after it.